### DISCUSSION TITLE
How do I test these APIs, Where are they hosted?

### DISCUSSION THREAD

**Discussion by deepanshurathi553** [Open-network-for-digital-commerce/ONDC-Protocol-Specs#38] [Q&A]
I want to test these APIs, but I couldn't find relevant documentation on where they are hosted, How do i test?

**Comment by shreyas707** [comment]
+1 Looking to test and understand how the APIs work. Any estimates on when it will be available would greatly help!

**Comment by rythmn1111** [comment]
Any one facing the same issue kindly mail me at rythmnmagnani0@gmail.com, don't worry i will guide you.

**Reply by Gokul-G22** [reply]
thank you bro we are in staging registry we finish our process sigining and verfication process we going to create api for the onsearch ,search and others we still in confusion in where to start using can u assist me to finish the task bro
